### ISSUE TITLE
Run the JavaFX application, I see it showing up first at the top-left position of my screen before it is centered

### ISSUE THREAD

**Issue by lvxingtu** [javafxports/openjdk-jfx#217]
I use sdkman to install java. When I use 10.0.2-oracle version, its okay.But problem will happen when use 11.ea.28-open or 12.ea.12-open.My application is the official getting started case[https://openjfx.io/openjfx-docs/#install-javafx](url)
![_ _20180924112108](https://user-images.githubusercontent.com/22794227/45953125-ba89da00-c03b-11e8-9924-f6d6697536c0.gif)

**Environment Details**
OS:Manjaro 17.1.12 Hakoila
DE: GNOME

**Comment by lvxingtu** [comment]
my system: manjaro linux 
![default](https://user-images.githubusercontent.com/22794227/45957197-0d1cc380-c047-11e8-8251-955c973cd78e.png)

**Comment by lvxingtu** [comment]
My system:manjaro linux
![default](https://user-images.githubusercontent.com/22794227/45957338-5d942100-c047-11e8-98ff-8130a11ed49a.png)

**Comment by kevinrushforth** [comment]
Starting with JavaFX 11 we use GTK 3 by default. Can you try with the following option and see if it make a difference?

```    java -Djdk.gtk.version=2 ...```

**Comment by johanvos** [comment]
Add it in the build.arg, by passing it to the jvmArgs, e.g.

run {
    doFirst {
        jvmArgs = [
            '--module-path',
classpath.filter{!it.name.startsWith("javafx-") ||
it.name.contains("-$classifier")}.asPath,
            '--add-modules', 'javafx.controls','-Djdk.gtk.version=2',
'-Dprism.verbose=true'
        ]
        classpath = sourceSets.main.output
    }
}


Op di 25 sep. 2018 om 02:21 schreef lvxingtu <notifications@github.com>:

**Comment by johanvos** [comment]
Add it in the build.arg, by passing it to the jvmArgs, e.g.
```

run {
    doFirst {
        jvmArgs = [
            '--module-path', classpath.filter{!it.name.startsWith("javafx-") || it.name.contains("-$classifier")}.asPath,
            '--add-modules', 'javafx.controls','-Djdk.gtk.version=2', '-Dprism.verbose=true'
        ]
        classpath = sourceSets.main.output
    }
}
```

**Comment by lvxingtu** [comment]
Thank you very much.  @johanvos

**Comment by lvxingtu** [comment]
The problem did not occur when using '-Djdk.gtk.version=2' on my computer

**Comment by tsayao** [comment]
I can confirm this also happens on ubuntu 18.04 with X.org (not tested with wayland).
Setting -Djdk.gtk.version=2 fixes it, so it appears to be a gtk3 issue.

I noticed it also happens on the bundled oracle 10 javafx when setting -Djdk.gtk.version=3.

**Comment by kevinrushforth** [comment]
Fixed by #446
